packages/server-wallet/src/chain-service/mock-chain-service.ts

```diff
@@ -36,6 +36,10 @@ const mockTransactoinResponse: providers.TransactionResponse = {
 };
 
 export class MockChainService implements ChainServiceInterface {
+  async checkChainId(_networkChainId: number): Promise<void> {
+    // noop, a mock chain service will have the "correct" chain id
```

Comment:
Noop here as in no operation (labelling blank function)? Not suggesting a change, just testing my assumption.

Reply:
Yeah that's right